src/lib/spinel/CMakeLists.txt

```diff
@@ -31,13 +31,19 @@ add_library(openthread-spinel-rcp)
 
 target_compile_definitions(openthread-spinel-ncp PRIVATE
     OPENTHREAD_FTD=1
-    OPENTHREAD_CONFIG_NCP_HDLC_ENABLE=1
     PUBLIC OPENTHREAD_SPINEL_CONFIG_OPENTHREAD_MESSAGE_ENABLE=1
 )
 
+if (OT_NCP_SPI)
+    target_compile_definitions(openthread-spinel-ncp PRIVATE OPENTHREAD_CONFIG_NCP_HDLC_ENABLE=0)
```

Comment:
Any reason these lines are duplicated twice here?

Reply:
One is `**ncp**` and second is `**rcp**`.

Reply:
They're not notice ncp vs rcp

Reply:
@parag-silabs Yes, seems to be a typo then.  Needs change for one of the lines in both cases to say `rcp`

Reply:
My bad.